gremlin-javascript/src/main/javascript/gremlin-javascript/lib/process/traversal-strategy.js

```diff
@@ -69,7 +70,32 @@ class TraversalStrategy {
   }
 }
 
+class VertexProgramStrategy extends TraversalStrategy {
+
+  constructor(graphComputer, workers, persist, result, vertices, edges, configuration) {
+    super();
+    this.configuration = new Map();
+    if (graphComputer !== undefined)
+      this.configuration.set("graphComputer", graphComputer);
+    if (graphComputer !== undefined)
```

Comment:
I think this condition was meant to be `workers !== undefined`, the following conditions have the same problem.

Reply:
yes, that was a mistake. fixed that. thanks